src/main/java/com/openshift/internal/restclient/model/Pod.java

```diff
@@ -61,10 +74,38 @@ public String getHost() {
 		}
 		return images;
 	}
+	
+	/**
+	 * The logic of the method is a copied from 'podStatus' function of
+	 * [app/scripts/filters/resources.js] of [openshift/origin-web-console]
+	 */
 
 	@Override
 	public String getStatus() {
-		return asString(POD_STATUS);
+		if (get(POD_DELETION_TIMESTAMP).isDefined()) {
```

Comment:
I thought it existed in JbossDmrExtentions, but it is not, would be to use has(NODE_PATH) because get coupled with 'isdefined' actually creates the child node where 'has' only checks for it.  Depending on the urgency of the PR, I would suggest adding 'has(String...)' to the utility class